issue_metrics.py

```diff
@@ -67,7 +68,11 @@ def get_env_vars() -> tuple[str, str, List[str]]:
     else:
         ignore_users = []
 
-    return search_query, token, ignore_users
+    report_title = os.getenv("REPORT_TITLE")
+    if not report_title:
+        report_title = "Issue Metrics"
```

Comment:
Is this a good place to implement the default value?
Or would you rather set the default in the signature of te `write_to_markdown()` function?
Or elsewhere?

Reply:
agree this should probably go in the write_to_markdown signature